src/services/__mocks__/DataService.ts

```diff
@@ -0,0 +1,35 @@
+// TODO remove after moving to prettier
```

Comment:
is this comment still relevant?

Reply:
There was probably a eslint disable comment here previously. I'l clean this up